clients/lsp-volar.el

```diff
@@ -47,6 +47,12 @@
   :group 'lsp-volar
   :package-version '(lsp-mode . "9.0.0"))
 
+(defcustom lsp-volar-hybrid-mode nil
+  "Enable Hybrid Mode."
+  :type 'boolean
+  :group 'lsp-volar
+  :package-version '(lsp-mode . "9.0.0"))
```

Comment:
Let's move the version to the next version, 9.0.1.

Reply:
done